openlibrary/plugins/openlibrary/pages/config_edition.page

```diff
@@ -25,6 +37,42 @@
             "notes": "",
             "name": "finnish_public_libraries_classification_system",
             "label": "Finnish Public Libraries"
+        },
+        {
+            "website": "http://www.udcc.org/",
+            "notes": "",
+            "name": "udc",
+            "label": "Universal Decimal Classification"
+        },
+        {
+            "website": "http://www.shl.lon.ac.uk/library/servicesandfacilities/information/classmarks.shtml",
+            "notes": "",
+            "name": "ulrls_classmark",
+            "label": "ULRLS Classmark"
+        },
+        {
+            "website": "http://www.ub.uni-frankfurt.de/en/english.html",
+            "notes": "",
+            "name": "goethe_university_library,_frankfurt",
+            "label": "Goethe University Library, Frankfurt"
+        },
+        {
+            "website": "http://www.leren.nl/artikelen/2004/siso.html",
+            "notes": "",
+            "name": "siso",
+            "label": "SISO"
+        },
+        {
+            "website": "http://nl.wikipedia.org/wiki/Nederlandse_Uniforme_Rubrieksindeling",
+            "notes": "",
+            "name": "nur",
+            "label": "NUR"
+        },
+        {
+            "website": "http://www.iccu.sbn.it/opencms/opencms/it/main/sbn/ (in italian)",
+            "notes": "",
+            "name": "identificativo_sbn",
+            "label": "Identificativo SBN"
         }
     ],
     "roles": [
```

Comment:
These roles are interesting, but do we know if they are used anywhere? I thought that any roles fields were free text. Is this an example of a wishful thinking feature that never made it?